### ISSUE TITLE
Book covers must have alt tags

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1061]
Level A violation: Images must have alternate text.

Book covers in this context should be treated as decorative elements so a null alt tag is sufficient. Add `alt=""` to [book covers](https://github.com/pulibrary/orangelight/blob/d8bdd5d626fd7523f75580dc0248e6352625cd64/app/assets/javascripts/book_covers.js.coffee#L36) `img` tag.
